spec/unit/pretty_format_spec.rb

```diff
@@ -49,33 +89,44 @@ def method_missing(*args, &block)
       end
 
       it "formats it with the default long format" do
-        expect(pretty_format(t)).to eq "28 de febrero de 1985 20:15"
+        expect(formatted_obj).to eq "28 de febrero de 1985 20:15"
       end
 
       it "formats it with a customized long format" do
         with_translation time: { formats: { long: "El %d de %B de %Y a las %H horas y %M minutos" } } do
-          expect(pretty_format(t)).to eq "El 28 de febrero de 1985 a las 20 horas y 15 minutos"
+          expect(formatted_obj).to eq "El 28 de febrero de 1985 a las 20 horas y 15 minutos"
         end
       end
     end
   end
 
-  it_behaves_like 'a time-ish object', Time.utc(1985, "feb", 28, 20, 15, 1)
-  it_behaves_like 'a time-ish object', DateTime.new(1985, 2, 28, 20, 15, 1)
+  context 'when given a Time in utc' do
+    let(:obj) { Time.utc(1985, "feb", 28, 20, 15, 1) }
+
+    it_behaves_like 'a time-ish object'
+  end
+
+  context 'when given a DateTime' do
+    let(:obj) { DateTime.new(1985, 2, 28, 20, 15, 1) }
+
+    it_behaves_like 'a time-ish object'
+  end
 
   context "given an ActiveRecord object" do
+    let(:obj) { Post.new }
+
     it "should delegate to auto_link" do
-      post = Post.new
-      expect(self).to receive(:auto_link).with(post) { "model name" }
-      expect(pretty_format(post)).to eq "model name"
+      expect(view).to receive(:auto_link).with(obj).and_return("model name")
+      expect(formatted_obj).to eq "model name"
     end
   end
 
   context "given an arbitrary object" do
+    let(:obj)  { Class.new.new }
```

Comment:
This seemed odd at first glance. Just to confirm is the double `.new` on purpose? I see it was that way originally though.

Reply:
Ok cool I'll just leave this as is then and resolve conversation. Thanks!